src/BundleSource.ts

```diff
@@ -225,7 +225,8 @@ ${file.headerContent ? file.headerContent.join("\n") : ""}`);
             this.concat.add(null, "(FuseBox)");
         }
         if (this.context.sourceMapsProject || this.context.sourceMapsVendor) {
-            this.concat.add(null, `//# sourceMappingURL=${this.context.bundle.name}.js.map`);
+            let sourceName = /[^\/]*$/.exec(this.context.bundle.name)[0];
```

Comment:
In basic cases yes.
In this example already, you will have `server/bundle` and `client/app`
https://github.com/fuse-box/fuse-box-examples/blob/master/examples/server-client/fuse.js